save-backend/src/main/kotlin/org/cqfn/save/backend/repository/TestRepository.kt

```diff
@@ -3,14 +3,15 @@ package org.cqfn.save.backend.repository
 import org.cqfn.save.entities.Test
 import org.springframework.stereotype.Repository
 
+import java.util.Optional
+
 /**
  * Repository of tests
  */
 @Repository
+@Suppress("MISSING_KDOC_CLASS_ELEMENTS", "MISSING_KDOC_ON_FUNCTION")
 interface TestRepository : BaseEntityRepository<Test> {
-    /**
-     * @param hash
-     * @return Test was founded by hash
-     */
-    fun findByHash(hash: String): Test?
+    fun findByHashAndFilePath(hash: String, filePath: String): Optional<Test>
```

Comment:
This method is used only in one test, is it needed?

Reply:
No, removed and updated the test